### PR TITLE
fix crd upgrade error

### PR DESCRIPTION
**Pull request by yuswift**
Signed-off-by: yuswiftli <yuswiftli@yunify.com>

**What type of PR is this?**

/kind bug
/milestone 3.1.0

### CODE REVIEW

**Comment by yuswift** [comment]
fix #3708

File: config/crds/storage.kubesphere.io_provisionercapabilities.yaml [hunk 1]
```diff
@@ -14,18 +14,23 @@ spec:
...
     plural: provisionercapabilities
     singular: provisionercapability
-  scope: Namespaced
+  scope: Cluster
   versions:
   - name: v1alpha1
     schema:
       openAPIV3Schema:
-        description: ProvisionerCapability is the schema for the provisionercapability API
+        description: ProvisionerCapability is the schema for the provisionercapability
```
Comment (zryfish) [inline config/crds/storage.kubesphere.io_provisionercapabilities.yaml:22]: Is this word wrap allowed in YAML?

Comment (yuswift) [reply config/crds/storage.kubesphere.io_provisionercapabilities.yaml:32]: yes

**Pull request merged**